asiignment8/front/data.json

```diff
@@ -0,0 +1,46 @@
+[{
```

Comment:
similar comments as in template 7 and please use es6 features, issues with readability and code redundancy